localMatchApp/headers/Login.h

```diff
@@ -0,0 +1,31 @@
+#ifndef LOGIN_H
+#define LOGIN_H
+
+#include <string>
+class Login {
+ public:
+  bool hasACapital ( const std::string i_password );
+  bool hasASmallLetter ( const std::string i_password );
+  bool hasANumber ( const std::string i_password );
+  bool hasCorrectFormat ( const std::string i_password );
+
+  enum class connectionMessage {
+    notEqualPasswords,
+    goodEntry,
+    doesntExists,
+    ocupatedUser,
+    wrongPassword
+  };
+
+ public:
+  Login ( );
+  ~Login ( );
+  Login ( const Login& ) = default;
+  Login& operator= ( const Login& ) = default;
+
+  connectionMessage signUp ( const std::string i_user, const std::string i_password,
```

Comment:
missing documentation for all these functions and variables of .h file

Reply:
Done.